### PR TITLE
Bug: listening first verse does not continue to the next.

### PR DESCRIPTION
**Pull request by AhmedShab**
The state of the duration was never set to 1. Therefore, sometimes, it was calculating the progress bar as `NaN` value, thus, the state never changed to call the next verse.

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32854

File: src/config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const config = {
   sentryServer: process.env.SENTRY_KEY_SERVER,
   facebookAppId: process.env.FACEBOOK_APP_ID,
   // Supported locales
-  locales: { en: 'English', ar: 'العربية', ur: 'اردو', id: 'Bahasa Indonesia', tr: 'Türkçe', fr: 'Français' },
+  locales: { en: 'English', ar: 'العربية', ur: 'اردو', id: 'Bahasa Indonesia', tr: 'Türkçe', fr: 'Langue du Site' },
```
Comment (AhmedShab) [inline src/config.js]: Not sure why this changed

File: src/redux/modules/audioplayer.js [hunk 2]
```diff
@@ -15,9 +15,9 @@ import {
   TOGGLE_SCROLL,
   UPDATE,
   // LOAD,
-  LOAD_SUCCESS,
+  LOAD_SUCCESS
   // LOAD_FAIL
-  } from 'redux/constants/audioplayer.js';
+} from 'redux/constants/audioplayer.js';
```
Comment (AhmedShab) [inline src/redux/modules/audioplayer.js]: Nor these two.

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32855

File: src/config.js (hunk 1 above)
Comment (sabeurthabti) [inline src/config.js]: hmm, try;
``git checkout src/config.js`

or 

`git checkout master -- src/config.js`

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32856

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32857

**Comment by AhmedShab** [comment]
Any feedback for this pull request? Thank you

**Comment by mmahalwy** [comment]
@AhmedShab sorry for being late. Is this the only big change? https://github.com/quran/quran.com-frontend/pull/771/files#diff-1a6763b6756935a86cbd6274219e82bcR36

**Comment by AhmedShab** [comment]
@mmahalwy sorry for my late respond as well, yes that's it. We never set duration to one before. Sometimes it was getting funny values like `NaN` and that's why you see the progress bar starts from the end or does something strange. It was probably dividing 0 with a number which makes it not a number.

What do you think?

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32782

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32783

**Comment by sabeurthabti** [comment]
@AhmedShab sorry for any delay approved. 

@mmahalwy all cool?

**Comment by sabeurthabti** [comment]
Thank you @AhmedShab for your contribution. Sorry for any delays in getting this merged in. I have tested it and the bug has been resolved. <3

**Comment by AhmedShab** [comment]
No worries 😄. Now I have a better idea about the code base, I will keep helping with this whenever I can. Most weekend I have more time, will do it during the week too when I have the chance 👍

**Pull request merged**